client/app/widgets/Notifications/index.jsx

```diff
@@ -45,90 +46,76 @@ export class Notifications extends React.Component<Props, State> {
     channel.bind('new_notification', (response: any) => {
       if (response && response.data) {
-        this.fetchNotifications();
+        fetchNotifications();
       }
     });
   };
 
-  setBody = (notifications: string[]) => {
+  const setBody = (notifications: string[]) => {
     let updatedNotifications = '';
     notifications.forEach((item: string) => {
       updatedNotifications += `<div>${item}</div>`;
     });
-    this.setState({ notifications: updatedNotifications });
+    setNotifications(updatedNotifications);
   };
 
-  fetchNotifications = () => {
-    const { alreadyMounted, signedInKey } = this.state;
+  const fetchNotifications = (): void => {
+    setAlreadyMounted(alreadyMounted);
+    setSignedInKey(signedInKey);
     return axios
       .get('/notifications/signed_in')
       .then((response: any) => {
         if (response && response.data && response.data.signed_in !== -1) {
           if (response.data.signed_in !== signedInKey) {
-            this.getPusherKey(response.data.signed_in);
+            getPusherKey(response.data.signed_in);
           }
           return axios.get('/notifications/fetch_notifications');
         }
         return -1;
       })
       .then((response: any) => {
         if (response && response.data && response.data.fetch_notifications) {
-          this.changeTitle(response.data.fetch_notifications.length);
-          this.setBody(response.data.fetch_notifications);
+          changeTitle(response.data.fetch_notifications.length);
+          setBody(response.data.fetch_notifications);
           if (!alreadyMounted && response.data.fetch_notifications.length > 0) {
-            this.setState({
-              alreadyMounted: true,
-              open: true,
-              modalKey: Utils.randomString(),
-            });
+            setAlreadyMounted(true);
+            setOpen(true);
+            setModalKey(Utils.randomString());
+            }
           }
-        }
       });
   };
 
-  clearNotifications = () => {
+  const clearNotifications = (): void => {
     axios.delete('/notifications/clear').then((response: any) => {
       if (response) {
-        this.changeTitle(0);
-        this.setState({ open: false, modalKey: Utils.randomString() });
-        this.fetchNotifications();
+        changeTitle(0);
+        setOpen(false);
+        setModalKey(Utils.randomString());
+        fetchNotifications();
       }
     });
   };
 
-  displayNotifications = () => {
-    const { notifications } = this.state;
+  const displayNotifications = (): void => {
+    setNotifications(notifications);
     return (
       <div aria-live="polite">
         {renderHTML(notifications)}
         <button
           type="button"
           className="buttonDarkS smallMarginTop"
-          onClick={this.clearNotifications}
+          onClick={clearNotifications}
         >
           {I18n.t('notifications.clear')}
         </button>
       </div>
     );
   };
-
-  render() {
```

Comment:
We shouldn't be deleting the return value of this. Instead of using the render function, we should just return.